src/InputContainer.test.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import { fireEvent, render } from '@testing-library/react';
+
+import InputContainer from './InputContainer';
+
+jest.mock('react-redux');
+
+describe('InputContainer', () => {
+  const dispatch = jest.fn();
+  beforeEach(() => jest.clearAllMocks());
+
+  useDispatch.mockImplementation(() => dispatch);
+  useSelector.mockImplementation((selector) => selector({
+    taskTitle: '할 일1',
+  }));
+
+  it('입력된 값과 element들을 출력합니다.', () => {
+    const { getByText, getByLabelText, getByPlaceholderText } = render((
+      <InputContainer />
+    ));
+
+    expect(getByText(/추가/)).not.toBeNull();
+    expect(getByLabelText('할 일')).not.toBeNull();
+    expect(getByPlaceholderText('할 일을 입력해 주세요')).not.toBeNull();
+
+    expect(getByText('할 일1')).not.toBeNull();
+  });
+
+  it('일정을 추가하는 함수가 실행합니다.', () => {
+    const { getByText } = render((
+      <InputContainer />
+    ));
+
+    expect(dispatch).not.toBeCalled();
+    fireEvent.click(getByText('추가'));
+    expect(dispatch).toBeCalled();
```

Comment:
```suggestion
    expect(dispatch).not.toBeCalled();
    
    fireEvent.click(getByText('추가'));
    
    expect(dispatch).toBeCalled();
```

의미상으로 구분되는 경우 빈 줄을 넣어서 구분해주면 좋아요.

Reply:
빈 줄을 추가해서 의미상 구분하겠습니다.